### ISSUE TITLE
Rescatux 0.71-beta7 radeon issue (continuation)

### ISSUE THREAD

**Issue by cjg67** [rescatux/rescatux#99]
Hello Adrian,

I have had no luck with getting my bios to boot from usb, but I tried again today in qemu, which I have only just installed.

1)  debian-live-10.0.0-amd64-lxqt.iso 
Again the startx command is needed to be issued before anything works.  After issuing startx the desktop comes up and all menu items are fully readable and it has had the correct screen resolution added.

2) debian-live-10.0.0-amd64-lxde.iso
This boots without any issues as well, does not need the startx to be typed on the command line.
Again the menu items are fully readable and everything appears to work fine.

I believe there should be no difference between using qemu and either a dvd rom drive or booting from a usb stick.

I tested each with:

qemu-system-x86_64 -boot d -cdrom debian-live-10.0.0-amd64-lxqt.iso -m 2g
qemu-system-x86_64 -boot d -cdrom debian-live-10.0.0-amd64-lxde.iso -m 2g

respectively.

I just could not leave the testing of it as I thought I could, it just bugs me.

**Comment by adrian15** [comment]
1) That's very strange that lxqt version does not boot right in qemu.
2) So I tested it. And you are right. These seems like a bug to me.
3) You should open a bug with reportbug against debian-live virtual package.
4) I have also tried: https://cdimage.debian.org/cdimage/weekly-live-builds/amd64/iso-hybrid/debian-live-testing-amd64-lxqt.iso and it does not work.
5) If you don't want to mess up too much with Rescatux I think you could use that lxde version that works for you and then use Rescapp on top of it. That way you can help with Rescapp installation documentation.

http://rescatux.sourceforge.net/repo/

(Well, the rescapp version for amd64 is quite old but I guess it could work.)

6) Anyway report the bug about lxqt version of Debian Live not working properly.
If you don't report the bug Debian won't know about it and it won't probably be fixed or looked at.

Thank you.

**Comment by adrian15** [comment]
> I tested each with:
> 
> ```
> qemu-system-x86_64 -boot d -cdrom debian-live-10.0.0-amd64-lxqt.iso -m 2g
> qemu-system-x86_64 -boot d -cdrom debian-live-10.0.0-amd64-lxde.iso -m 2g
> ```

Can you please test:


```
qemu-system-x86_64 -boot d -vga qxl -cdrom debian-live-10.0.0-amd64-lxqt.iso -m 2g
qemu-system-x86_64 -boot d -vga qxl -cdrom debian-live-10.0.0-amd64-lxde.iso -m 2g
```

instead and tell us how it goes?

Thanks.

**Comment by adrian15** [comment]
> I have had no luck with getting my bios to boot from usb

Are you using etcher or another tool?

**Comment by agaida** [comment]
so - after a few tests it boils down to sddm with -vga stdio - a small test to reproduce:
```
apt update
apt install slim
sudo slim
# hit f1 - lxqt session is the only session
# user= user
# pass= live
```
voila, one should make the screen a bit bigger - it works fine :D

**Comment by agaida** [comment]
![](https://i.imgur.com/XXdF1KT.png) - to be honest, nobody want the stdio driver to day - just horrible

**Comment by cjg67** [comment]
> > I tested each with:
> > ```
> > qemu-system-x86_64 -boot d -cdrom debian-live-10.0.0-amd64-lxqt.iso -m 2g
> > qemu-system-x86_64 -boot d -cdrom debian-live-10.0.0-amd64-lxde.iso -m 2g
> > ```
> 
> Can you please test:
> 
> ```
> qemu-system-x86_64 -boot d -vga qxl -cdrom debian-live-10.0.0-amd64-lxqt.iso -m 2g
> qemu-system-x86_64 -boot d -vga qxl -cdrom debian-live-10.0.0-amd64-lxde.iso -m 2g
> ```
> 
> or
> 
> ```
> qemu-system-x86_64 -boot d -vga virtio -cdrom debian-live-10.0.0-amd64-lxqt.iso -m 2g
> qemu-system-x86_64 -boot d -vga virtio -cdrom debian-live-10.0.0-amd64-lxde.iso -m 2g
> ```
> 
> instead and tell us how it goes?
> 
> Thanks.

Hello Adrian,

For this one:

qemu-system-x86_64 -boot d -vga qxl -cdrom debian-live-10.0.0-amd64-lxqt.iso -m 2g

It boots fine without needing to issue startx.  The loading time is way too long, as it takes 11 minutes to have a fully working desktop.  The menu items are all clear, as it has the correct resolution.

For this one:

qemu-system-x86_64 -boot d -vga virtio -cdrom debian-live-10.0.0-amd64-lxqt.iso -m 2g
Again it boots fine, without the need to issue startx, but again the loading time is way too long, at 9 minutes.

For this one:

qemu-system-x86_64 -boot d -vga virtio -cdrom debian-live-10.0.0-amd64-lxde.iso -m 2g

Again it boots fine, but again the loading time is way too long at 11 minutes.

The menu items are clearly readable.

The issue with my bios is that the usb stick, even after selecting boot from usb-hdd first, is not displayed in the list, no matter what I do.  I used the dd method to create the bootable drive.

Regards,

Christopher.

**Comment by agaida** [comment]
@cjg67 - one likely need the kvm accelerator if using qemu-system-x86_64 as in
```
qemu-system-x86_64 -machine q35,accel=kvm ...
```
Test it, you will notice the difference  :sunglasses:

**Comment by cjg67** [comment]
Hello,

Enabling acceleration was what was needed.  I had to add -enable-kvm at the start, and now bootup is quick.

Regards,

Christopher.

**Comment by cjg67** [comment]
Hello,

I just downloaded the latest lxqt version of the iso from:

https://cdimage.debian.org/cdimage/weekly-live-builds/amd64/iso-hybrid/
and used the following to test:

qemu-system-x86_64 -enable-kvm -boot d -vga virtio -cdrom debian-live-testing-amd64-lxqt.iso -m 2g

This cleanly boots and no startx is required.  This iso was from 2nd of this month.

Regards,

Christopher.

**Comment by cjg67** [comment]
Hello,

I just did a test of rescatux-0.71-beta7.iso in qemu and it boots cleanly on auto, and the screen resolution is perfect.  The menu items are fully readable, you have the full set of resolutions to change to/from.

I thought that I was going nuts, as I knew the errors that I reported did happen, so I managed to find my last blank cd and burnt a copy.

When I booted from the cdrom drive, then the errors that I mentioned in the first bug report happen.  You have to issue sudo startx, and when it does boot the menu items are unreadable, the screen resolution is unusable you can't change the resolution.

This raises the question, why does it work in qemu and not on either the usb-stick (when it was usable by bios) or a brand new cd?

I use brasero for burning, and it is a normal burn image to disk option that I use.

Regards,

Christopher.

**Comment by cjg67** [comment]
Hello,

Well I have managed to get that usb-stick to boot again.  It seems that with this frigging bios, you need to have the usb-stick in ONE particular port to boot.

So I used dd to copy the image of debian-live-10.0.0-amd64-lxqt.iso to the usb-stick and booted it.  I had to manually type startx to get it to work.  When it was booted there are only three resolutions that can be selected.  In qemu it boots cleanly without having to do anything and you have a large list of resolutions to select from.

I have never seen this type of weirdness before.

I am at a total loss as to why this is happening.  Is it a debian iso creation problem?  Some weird permission error that only occurs when copying it to the usb-stick or what?

Regards,

Christopher.

**Comment by agaida** [comment]
it depends - we should know more about your machine - the output of `inxi -SCIG``would help a lot.

**Comment by cjg67** [comment]
Hello,

Here is the requested info:

System:    Host: lfs Kernel: 5.0.17 x86_64 bits: 64 Desktop: Gnome 3.32.1 Distro: Linux From Scratch 20190511-systemd 
Machine:   Type: Desktop Mobo: Gigabyte model: GA-78LMT-USB3 6.0 v: SEx serial: N/A BIOS: Award v: F2 date: 11/25/2014 
CPU:       Topology: 6-Core model: AMD FX-6300 bits: 64 type: MCP L2 cache: 2048 KiB 
           Speed: 3770 MHz min/max: 1400/3500 MHz Core speeds (MHz): 1: 3478 2: 3083 3: 3785 4: 3861 5: 3748 6: 3268 
Graphics:  Device-1: Advanced Micro Devices [AMD/ATI] RS780L [Radeon 3000] driver: radeon v: kernel 
           Display: server: X.Org 1.20.4 driver: ati,radeon unloaded: fbdev,modesetting resolution: 1920x1080~60Hz 
           OpenGL: renderer: AMD RS780 (DRM 2.50.0 / 5.0.17 LLVM 8.0.0) v: 3.3 Mesa 19.0.4 
Info:      Processes: 198 Uptime: 28m Memory: 7.27 GiB used: 1.62 GiB (22.3%) Shell: bash inxi: 3.0.36

**Comment by adrian15** [comment]
I'll add that this issue was a continuation of: https://github.com/rescatux/rescatux/issues/98#issuecomment-525418299

This is what I inferred from his tests against his physical machine:

> * Normal debian-live-testing-amd64-lxqt.iso : Needs startx so that desktop is shown.
> * Non-free debian-live-testing-amd64-lxqt.iso : It will not get past verifying the drm pool
> * Normal debian-live-testing-amd64-lxde.iso : Boots into desktop ok.
> * Non-free debian-live-testing-amd64-lxde.iso : Not tested.

The debian-live-testing-amd64-lxqt.iso : Needs startx so that desktop is shown seems to be confirmed.

**Comment by agaida** [comment]
Conclusion: Please file a bug against sddm - and get it rightfully rejected
> Graphics: Device-1: Advanced Micro Devices [AMD/ATI] RS780L [Radeon 3000] driver: radeon v: kernel
Radeon need firmware-non-free

**Comment by adrian15** [comment]
I was reading: https://bugs.launchpad.net/ubuntu/+source/sddm/+bug/1431332 .
1) @cjg67 : Can you run:
```
systemctl status sddm
```
when you get the text-only prompt?

What's the output from that?

2) And finally can you run:
```
systemctl status sddm
```
?

Does the graphical desktop appear?

**Comment by adrian15** [comment]
> Radeon need firmware-non-free

Well, @cjg67 needs to confirm it but according to his old tests the need of non-free firmware shouldn't be a problem because of this:

> Normal debian-live-testing-amd64-lxde.iso : Boots into desktop ok.

Hopefully he can dd that image again to an usb and test it again so that we can discard the need for radeon firmware.

**Comment by agaida** [comment]
it is a problem for sddm - and only a wild guess - it will be a problem for gdm too not to have not more than very basic graphic features available - to be honest right now i'm not exactly know which part is is - gl or some kind of acceleration

**Comment by cjg67** [comment]
Hello,

Well I booted debian-live-10.0.0-amd64-lxqt again, and after it dropped to txt mode I ran systemctl status sddm, and it said it was active and running.  I used systemctl restart sddm and the desktop did not appear until I issued startx.  Then as stated above, there is only three options to choose from for the resolution.

I did sudo journalctl -r and noticed a reference to no caching page file found.

I will use dd to test another image.  

The interesting thing is, on this computer I have sddm, gdm and lxdm installed and they all function correctly.

Regards,

Christopher.

**Comment by cjg67** [comment]
Hello,

I just used dd to copy debian-live-10.0.0-amd64-lxqt+nonfree.iso to the usb-stick.

When I booted, it was a clean boot, all the screen resolutions are present and all text is fully readable   
with perfect clarity of the images.

As Adrian pointed out above, this test was not done before.  I was trying to see if it was usable without the non-free blobs in the kernel.

It is still doing my head in as to why rescatux works in a VM and not on the physical hardware.  It seems that the key is to find out what the difference is, so that it can be narrowed down to the point of perhaps actually being able to solve it.

I do not usually use a VM for testing things, as I have always suspected that there are differences with how things are run.  

Regards,

Christopher.

**Comment by cjg67** [comment]
Hello,

I just used dd to copy debian-live-testing-amd64-lxqt.iso to the usb-stick.  

With this, again you get dropped into txt mode and have to issue the startx command.  Restarting sddm does not bring up the desktop.

However with this iso, you can not get into the monitor settings, as clicking on it does nothing.  The power-off menu also is inoperative.  The screen resolution is wrong, just by the overall appearance.

Regards,

Christopher.

**Comment by cjg67** [comment]
Hello,

At this stage of things, the best approach with perhaps the least amount of work would be to have two versions of the rescatux iso.  The current one, which seems to work for most, and then build a second one that has the non-free for those who have a radeon card that needs the non-free.

The reason that I make this suggestion, is that for some reason the lxqt version that has the non-free added works out of the box.  It could well be that as it is an unofficial version, that whomever made it has made other changes.  

I guess that one approach could be to try and reverse engineer it, a bit at a time.  Start by removing the non-free and see if it still has the issues when booting from usb-stick.  If it does not, then they may well have made some other changes.

Regards,

Christopher.

**Comment by adrian15** [comment]
> Well I booted debian-live-10.0.0-amd64-lxqt again, and after it dropped to txt mode I ran systemctl status sddm, and it said it was active and running. I used systemctl restart sddm and the desktop did not appear until I issued startx. Then as stated above, there is only three options to choose from for the resolution.

What are those three resolutions? I have an idea.

Thank you.

**Comment by cjg67** [comment]
Hello Adrian,

I am sorry but debian-live-10.0.0-amd64-lxqt that I was talking about there was on the VM and not on the physical hardware.  On the physical hardware I am unable to even get into the monitor settings and the shut-down menu is also inoperable.

Regards,

Christopher.

**Comment by cjg67** [comment]
Hello Adrian,

I just got delivered a box of re-writable dvd's, so I decided to burn a copy of debian-live-10.0.0-amd64-lxqt on it to test.  

It still boots to the txt only screen, however, now the monitor settings are accessible, and the power off menu etc is operational.

It still only has three settings for the resolution which are:

720 x 480
640 x 480
720 x 400

Regards,

Christopher.

**Comment by adrian15** [comment]
And what are the resolutions that you get with:
Normal debian-live-testing-amd64-lxde.iso
?

**Comment by cjg67** [comment]
Hello Adrian,

I downloaded debian-live-10.1.0-amd64-lxde.iso, burnt it to dvd and it dropped to console, and after issuing startx the desktop fully loaded, however it hard locks, ie no keyboard movement no mouse.

I then burnt debian-live-10.0.0-amd64-lxde.iso and again it booted to console, after issuing startx the desktop fully loaded and after going into monitor settings the resolutions available are exactly the same.

Regards,

Christopher.

**Comment by cjg67** [comment]
Hello Adrian,

I downloaded debian-live-10.1.0-amd64-lxqt.iso,  burnt it to dvd and again it dropped to console, and after issuing startx the desktop fully loaded, and once again it hard locks and I have to use the reset button.

Regards,

Christopher.

**Comment by cjg67** [comment]
Hello Adrian,

I just downloaded debian-live-testing-amd64-lxde.iso from the 2'nd of this month, burnt it to dvd and it boots to console.  After issuing startx, the desktop fully loads, and then there is once again an unusable keyboard and mouse, and I have to use the reset button.

Regards,

Christopher.

**Comment by cjg67** [comment]
Hello Adrian,

I downloaded debian-live-10.1.0-amd64-lxqt.iso from the 7th of this month from:

https://cdimage.debian.org/debian-cd/current-live/amd64/iso-hybrid/ 
and again it drops to console and after typing startx desktop fully loaded and again the keyboard and mouse are unusable and I have to use the reset button.

I wanted to report the bug to debian, but it seems that they have turned into prize dickheads with their bug tracker.  You have to bloody well use their ridiculous commands in an email for it to even get sent.  There is no way I am going to try to even work it out.  They do not want bugs submitted.

Regards,

Christopher.

**Comment by agaida** [comment]
please don't write any reports to debian - esp. not to LXQt - maybe not to sddm - they work all es expected. I mentioned it before, it might be that you only use the fallback video with your hardware - so please provide any informations about and don't write again and again what not work for you. A good start would be the output of `inxi -SGCMIxxx` - we need a base knowledge of your machine when we try to help - and please for god sake - if you run AMD, please use the non-free live isos. They will possible work far better with your hardware.

**Comment by cjg67** [comment]
Hello,
I am sorry but I find it very hard to beleave that they all work as expected when I download an iso from the so-called stable pool and the keyboard and mouse do not work.  The iso's behave differently when run in qemu.  Have you downloaded debian-live-testing-amd64-lxde.iso and burnt it to cd/dvd and tested to see if the mouse and keyboard work?  

I have both a usb keyboard and mouse.  All of the iso's from https://cdimage.debian.org/debian-cd/current-live/amd64/iso-hybrid/ are in a current state of being broken when they are run on physical hardware.

There are differences in running in a VM and on the physical machine, yet no one can seem to tell me the reason why this is so.  

Adrian asked me to test things, and specifically did not want to build rescatux with the non-free firmware.  I told Adrian in the previous thread that I was using an AMD chipset, and the radeon drivers do need the firmware installed, as I have the firmware for that built into my kernel on my computer, along with firmware for an INTEL based wifi network card that I just purchased.

I have tested everything and I have posted the findings.  I am now just going around in circles as debian break things and don't  give a damn.  I doubt that they even burn a copy and test it, just use it in a VM and then push it out.

Here is the output as requested, and this is for the second time within this thread:

System:    Host: lfs Kernel: 5.2.14 x86_64 bits: 64 compiler: gcc v: 9.1.0 Desktop: Gnome 3.32.1 wm: gnome-shell 
           dm: GDM 3.32.0 Distro: Linux From Scratch 20190511-systemd 
Machine:   Type: Desktop Mobo: Gigabyte model: GA-78LMT-USB3 6.0 v: SEx serial: N/A BIOS: Award v: F2 date: 11/25/2014 
CPU:       Topology: 6-Core model: AMD FX-6300 bits: 64 type: MCP arch: Bulldozer L2 cache: 2048 KiB 
           flags: avx lm nx pae sse sse2 sse3 sse4_1 sse4_2 sse4a ssse3 svm bogomips: 42184 
           Speed: 1465 MHz min/max: 1400/3500 MHz boost: enabled Core speeds (MHz): 1: 2303 2: 2449 3: 3306 4: 2582 5: 2627 
           6: 3558 
Graphics:  Device-1: Advanced Micro Devices [AMD/ATI] RS780L [Radeon 3000] vendor: Gigabyte driver: radeon v: kernel 
           bus ID: 01:05.0 chip ID: 1002:9616 
           Display: server: X.Org 1.20.4 driver: ati,radeon unloaded: fbdev,modesetting alternate: vesa 
           compositor: gnome-shell resolution: 1920x1080~60Hz 
           OpenGL: renderer: AMD RS780 (DRM 2.50.0 / 5.2.14 LLVM 8.0.0) v: 3.3 Mesa 19.0.4 compat-v: 3.0 direct render: Yes 
Info:      Processes: 201 Uptime: 2h 56m Memory: 7.27 GiB used: 3.61 GiB (49.6%) Init: systemd v: 241 runlevel: 5 Compilers: 
           gcc: 9.1.0 clang: 8.0.0 Shell: bash (su) v: 5.0.0 running in: gnome-terminal inxi: 3.0.36

**Comment by agaida** [comment]
As a matter of fact: The so called "free" radeon driver need non-free firmware to work  - like it or not, i'm ranting about it for the last few years. Anyways, ranting will not change a thing. And hardware has no feelings about right or wrong or political correctness regarding needed things  - without it will just refuse to work if needed parts are missed. If you want your hw to work, just try the so called non-free "firmware-iso" from debian.

 https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/10.1.0-live+nonfree/

Next wrong assumtion - "Debian" don't break anything - it was not their idea to depend on non-free firmware. To put it short, we all know that non-free firmware is needed for radeon to work correctly - and we all know too that this firmware is non-free and can't be on an image that only contain free software. It is really that easy. There are discussion if firmware is that kind of software where the non-free rules apply: Imho not, other people think otherwise - it's a very political and philosophical question.

In short: A project is allowed to distribute the firmware blobs that are on the non-free iso - but these blobs will not provide the four software freedoms defined by the rms and the fsf - so they can't be on the free iso according to the Debian policy.

**Comment by cjg67** [comment]
Hello,
Debian have broken things in the past.  Debian bowed down to the trolls that red-hat sent out to push their crap of selinux.  I know this for a FACT.  I saw the trolls with redhat email addresses spout on the forums.  I am talking 10 or more years ago.  
I have been using linx/unix for decades, and have worked as system administrator for webhosting companies and done support work for isp's.

I know a BROKEN iso when I see one.  You are incorrect with regards to the free radeon driver, as I have proven that it DOES work with my hardware.

I thought that Adrian was the maintainer of rescatux, am I incorrect in this?

I did not expect to have another person here going against what Adrian wants, nor to have an argument about it.

What I thought this testing was about was to get a version of rescatux that works with the non-free software WITHOUT the firmware.

It seems though, that this is not about getting rescatux working on any of the versions of debian that I have proved work, but just to get me to test this iso or that, and then have you come and spout about testing something THAT I DO NOT NEED TO.

I build ALL my os's now from source code.  I know the hardware works. 

Please advise me if you are the one who has taken over rescatux.

I am NOT going to test non-free debian.  I have wasted enough band-width.

I am a trained, qualified IT professional.

I am now very angry with you.

**Comment by agaida** [comment]
ok, you "pro" - i'm "only" upstream for LXQt, Debian maintainer for LXQt and tried to teach you that some things like sddm likely need some more from the graphics stack than the GTK based things - anyways, if you are angry with me, so it may be - don't care about. The only relation to rescatux is that adrian now uses LXQt. Thats all. So just go ahead and do what you want to - only one thing, don't spread false claims - that makes me angry and harsh. At least you were not very helpful  - but thank you for pointing out that sddm __might__ be the wrong DM.

@adrian15 - it might be a good idea for rescatux to use lightdm instead of sddm in future releases - the LXQt packaging is prepared for, so if you put lightdm into your package list and configure it a bit the problem should be solved.
```
Depends: ${misc:Depends},
        ....
        sddm-theme-debian-elarun [linux-any] | sddm-theme [linux-any] | lightdm
         | gdm3 [linux-any] | lxdm [linux-any] | slim | nodm [linux-any],
        ...
```

**Comment by cjg67** [comment]
Hello,
So you refuse to burn something to dvd to test. And just spout that it all works, which is a complete LIE.
You are obviously not professional in anything.  I beleave what I see with the proof of my eyes.
Saying that I was not helpful is pure BULLSHIT.  Fix the broken shit that you call iso's on the debian servers if you are really a maintainer.

**Comment by agaida** [comment]
Your opinion - but hey - you are wrong. Nevermind. Anyways, i pointed out a possible solution - any you might be surprised: Yes, sddm need some things that GTK don't need - so i recommend to switch to LightDM as a tool that better fit for rescatux - ymmv, but the SDDM devs will not change a line because of your complains.

**Comment by adrian15** [comment]
> debian-live-10.0.0-amd64-lxqt on one to test.
>
> 720 x 480
> 640 x 480
> 720 x 400
> (...)

> I then burnt debian-live-10.0.0-amd64-lxde.iso and again it booted to console,
> after issuing startx the desktop fully loaded and after going into monitor
> settings the resolutions available are exactly the same.

So that means that either sddm or lightdm in your system only offer:
 * 720 x 480
 * 640 x 480
 * 720 x 400
.

Rescapp needs 1024x768 resolution to be usable.
So you are going to need the non-free iso so that it loads 1024x768 resolution in your system.


The historic reason for trying to blame on sddm or even thinking on adding lightdm to Rescatux was that supposedly one of your early normal LXDE live boot brought out graphical interface ok for you.

Now I'm suspecting that it wasn't the case and that you were just running the non-free LXDE live.

So, afterall, @agaida which has more experience than me was right with his first intuition on you using non-free so that radeon firmware is used.

**Comment by adrian15** [comment]
@cjg67 : Can you please open a new issue titled 'Install Rescapp on non-free Debian Live LXQT Buster' ?
The current process involves adding a new repo and the current amd64 version is outdated compared to the i386 version. So the i386 arch would have to be added.
I mean... I have to write an script that you can just copy and paste in the command line just a line and that installs it for the average user.

And I need a new issue so that I don't forget about it.

Thank you.

**Comment by adrian15** [comment]
> I did not expect to have another person here going against
> what Adrian wants, nor to have an argument about it.
I asked @agaida on irc on this specific issue about resolution because Debian LXDE based live cd seemed to work in your system while the Debian LXQT based live cd did not work. He maintains lxqt package at Debian that's why I asked him.

As I said before @agaida seems to have nailed from the start what your specific problem was. Not having the proper non-free firmware for radeon. I was more optimistic than him. I thought we were able to find a workaround.

Now that you have proven that the biggest resolution that you get with LXDE based live cd is less than 1024x768 (and that's needed for rescapp to work properly) I'm discarding this path.

> What I thought this testing was about was to get a version
> of rescatux that works with the non-free software WITHOUT the firmware.

Yeah, as I say, I'm discarding this path. If I ever tweaked Rescatux to be built with lightdm instead of sddm you might get into a 720x480 desktop and that's not usable for Rescapp.

**Comment by adrian15** [comment]
> @adrian15 - it might be a good idea for rescatux to use lightdm
> instead of sddm in future releases - the LXQt packaging is prepared for,
> so if you put lightdm into your package list and configure it a bit the problem should be solved.

As I have discussed in previous comments I don't think it's worth switching to lightdm if no 1024x768 resolution is won at all.

Thank you for the advice. I was also actually thinking that switching to lightdm was going to be the solution.

**Comment by adrian15** [comment]
As I think there is nothing more to discuss here I'll close the issue.
@agaida : Thank you for your valuable help.
@cjg67 : Please open the bug I requested you so that I can improve the Rescatux documentation and avoid new users your experience. Thank you for all the tests you have done.
Learning how Radeon cards work has been valuable.

**Comment by agaida** [comment]
It will not help with rescatux - if the resolution is not available - that much is true - but the user would be in the graphical environment - and maybe it is easier to install the needed stuff from there - only an idea. With my LXQt hat on: We are not limited to any kind of DM - maybe in case of rescatux even `nodm` will be a sufficient DM (not a joke :smile)

**Comment by adrian15** [comment]
Well, even if the resolution is not available it also bothers me that you have to type startx even in LXDE for graphical environment to show up.

Yeah, I don't want to switch to lightdm to avoid it fetching extra gtk libraries. And I don't want to switch to nodm in order to minimize my modifications to raw lxqt meta-package (even if I already have messed up with that lxqt meta-package ;) ).

**Comment by agaida** [comment]
```
Depends: ${misc:Depends},
        ....
        sddm-theme-debian-elarun [linux-any] | sddm-theme [linux-any] | lightdm
         | gdm3 [linux-any] | lxdm [linux-any] | slim | nodm [linux-any],
        ...
``` 
in case of DMs you only has to introduce the whished package - it is ok with the metapackages - default alternative would be sddm-$foo, but everything that provide `x-display-manager` should work fine. Thats also true for the WM and other parts of the meta-packages - and please - if you miss something - give me a ping.
